components/admin_console/admin_sidebar/admin_sidebar.tsx

```diff
@@ -208,7 +222,7 @@ class AdminSidebar extends React.PureComponent {
                         }
                     }
                     const subDefinitionKey = `${key}.${subKey}`;
-                    let tag = '';
+                    let tag: string | JSX.Element = '';
```

Comment:
Sry, haven't seen this the first time around:

```suggestion
                    const tag = item.tag?.shouldDisplay(license) ? item.tag.value : '';
```

and then remove the if-clause below

Reply:
Yes, you are right. I missed it as well 👍